azext_edge/edge/providers/check/dataflow.py

```diff
@@ -0,0 +1,1228 @@
+    )
+    # show dataflow endpoint ref on detail
+    if detail_level > ResourceOutputDetailLevel.summary.value:
+        padding += PADDING_SIZE
+        endpoint_name_display = f"{{{colorize_string(value=endpoint_ref)}}}"
+        endpoint_validity_display = colorize_string(color=endpoint_status.color, value=endpoint_validity)
+        check_manager.add_display(
+            target_name=target,
+            namespace=namespace,
+            display=Padding(
+                f"Dataflow Endpoint {endpoint_name_display} {endpoint_validity_display}",
+                (0, 0, 0, padding),
+            ),
+        )
+    elif not endpoint_match:
+        check_manager.add_display(
+            target_name=target,
+            namespace=namespace,
+            display=Padding("[red]Invalid destination endpoint reference[/red]", (0, 0, 0, padding - PADDING_SIZE)),
+        )
+    # only show destination on verbose
+    if detail_level > ResourceOutputDetailLevel.detail.value:
+        for label, key in [
+            ("Data Destination", "dataDestination"),
+        ]:
+            val = settings.get(key)
+            if val:
+                check_manager.add_display(
+                    target_name=target,
+                    namespace=namespace,
+                    display=basic_property_display(label=label, value=val, padding=padding),
+                )
+
+
+def _process_endpoint_mqttsettings(
+    check_manager: CheckManager, target: str, namespace: str, spec: dict, detail_level: int, padding: int
+) -> None:
+    settings = spec.get("mqttSettings", {})
+    for label, key in [
+        ("MQTT Host", "host"),
+        ("Protocol", "protocol"),
+    ]:
+        val = settings.get(key)
+        if val:
+            check_manager.add_display(
+                target_name=target,
+                namespace=namespace,
+                display=basic_property_display(label=label, value=val, padding=padding),
+            )
+    if detail_level > ResourceOutputDetailLevel.detail.value:
+        for label, key in [
+            ("Client ID Prefix", "clientIdPrefix"),
+            ("Keep Alive (s)", "keepAliveSeconds"),
+            ("Max Inflight Messages", "maxInflightMessages"),
+            ("QOS", "qos"),
+            ("Retain", "retain"),
+            ("Session Expiry (s)", "sessionExpirySeconds"),
+        ]:
+            val = settings.get(key)
+            if val:
+                check_manager.add_display(
+                    target_name=target,
+                    namespace=namespace,
+                    display=basic_property_display(label=label, value=val, padding=padding),
+                )
+
+        tls = settings.get("tls", {})
+        check_manager.add_display(
+            target_name=target,
+            namespace=namespace,
+            display=Padding("TLS:", (0, 0, 0, padding)),
+        )
+        for label, key in [
+            ("Mode", "mode"),
```

Comment:
Is the value of Mode disable and enable? In dp pipeline when the mode is disabled I will skip the evalution if the pipeline is disabled since it's not used, wondering if this is the same case?🤔

Reply:
(just sharing some thought, but I think we can also think about this later after the release)